spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -85,7 +85,11 @@ def get_value(self, name):
             return self.call_kernel(
                 interrupt=True, blocking=True).get_value(name)
         except (TimeoutError, UnpicklingError):
-            raise ValueError('Kernel did not send the value.')
+            msg = _("The variable is either too big or not picklable."
+                    "<br><br>"
```

Comment:
Is there a way to know the difference between errors? (too big or not pickable?)

Reply:
Yes. TimeoutError is generated when the variable is too big and UnpicklingError when it's not picklable. I'll generate different messages for each one.